### DISCUSSION TITLE
property_mapping_exception, /plex/models.py (hundreds of error notifications)

### DISCUSSION THREAD

**Discussion by mgrimace** [goauthentik/authentik#11204] [Q&A] [answered]
Hello,

I woke up to 108 email notifications with the following error:
```Traceback (most recent call last): File "Overseerr Plex SSO", line 43, in <module> File "Overseerr Plex SSO", line 2, in handler builtins.ImportError: cannot import name 'PlexSourceConnection' from 'authentik.sources.plex.models' (/authentik/sources/plex/models.py)```

I haven't changed or done anything, any idea how to resolve this? Also, any reason why I'm getting hundreds of error notifications + emails about the error?

**Comment by morganorix** [comment]
Same problem... Since updated 2024.8.2, no supported external user. I've change user type to "internal". The scope not work.
I use principaly the sso plex... It is not possible to downgraded... thx
It's been 3 weeks, it would be nice to get moving.

**Comment by yandy-r** [comment, marked as answer]
I got rid of the import error by modifying the import.

```python
from authentik.sources.plex.models import UserPlexSourceConnection as PlexSourceConnection
```

I used the ```as``` statement so as not to change the rest of the code.

This got Overseerr working, though the Tautulli script doesn't work. I am curious to know if it did before as I'm just getting started and ran into this issue.

**Reply by mgrimace** [reply]
Thanks so much for sharing, where/how do you enter this?

**Reply by yandy-r** [reply]
Looks like you have a custom propery mapping for Overseer/Plex integration to login. Go to Admin Interface -> Customization -> Property Mappings and look for the Overseerr one. Click edit and under the mapping script, the first line in the import statement.

**Comment by yandy-r** [comment]
Fyi, Tautulli also works, my issue was just that I had a path ```/tautulli``` that needed to be accounted for in the ```base_url```.
